demo/Semi.Avalonia.Demo/Pages/AutoCompleteBoxDemo.axaml

```diff
@@ -30,7 +29,7 @@
 
         <AutoCompleteBox
             Watermark="Please select a State"
-            ValueMemberBinding="{Binding Name}" />
+            ValueMemberBinding="{Binding Name,DataType=vm:StateData}" />
         <AutoCompleteBox
             Classes="Large"
             ValueMemberBinding="{ReflectionBinding Name}" />
```

Comment:
All ReflectionBindings in this page should be updated to use Binding and DataType.